core/config/identifiers.json

```diff
@@ -29,6 +29,25 @@
       }
     }
   },
+  "CMC Total Market Cap": {
```

Comment:
Should we indicate somehow (in the name or somewhere else in the UI) that this scaling is happening?

Reply:
@hlambur wdyt?

Reply:
I think it's self explanatory enough... people should get it (and I can't think of a good way of describing this is < 10 chars). If they don't get it, we'll hear about it.

Reply:
SGTM